app/lib/shared/configuration.dart

```diff
@@ -49,6 +49,22 @@ const _fakeSiteAudience = 'fake-site-audience';
   disallowUnrecognizedKeys: true,
 )
 class Configuration {
+  /// The name of the Cloud Storage bucket to use for storing the uploaded
+  /// package archives.
+  ///
+  /// The bucket content policy should be private.
+  final String? canonicalPackagesBucketName;
+
+  /// The name of the Cloud Storage bucket to use for public package archive downloads.
+  ///
+  /// The bucket content policy should be public.
+  final String? publicPackagesBucketName;
+
+  /// The name of the Cloud Storage bucket to use for incoming package archives.
+  ///
```

Comment:
```suggestion
  ///
  /// When users are publishing packages using the `dart pub` client, they are given a signed-url
  /// which allows the client to upload a file. That signed-url points to an object in this bucket.
  /// Once the uploaded tarball have been verified, it can be copied to the canonical bucket.
```